lesson1/program1.js

```diff
@@ -0,0 +1,6 @@
+const myfunction= (a,b)=>{
+  c= a+b;
```

Comment:
`const c = ` always declare your variables with either a `const` or `let`
also, pick better variable names. `c` could be better renamed as `sum`